app/Global.scala

```diff
@@ -1,16 +0,0 @@
-import configuration.Config
```

Comment:
this has moved to `MyApplicationLoader.load` just below https://github.com/guardian/subscriptions-frontend/pull/1058/files#diff-751cafce06c7a60432eacdc822ece49dR39